mysql-test/t/backup_locks.test

```diff
@@ -554,48 +542,21 @@ CREATE DATABASE test1;
 
 --let $id=`SELECT CONNECTION_ID()`
 
---send CREATE PROCEDURE p1() SELECT 1
+--send INSERT INTO t_myisam VALUES(0)
```

Comment:
This is suspicious - it means that LOCK TABLES FOR BACKUP does not block CREATE PROCEDURE, which is DDL, anymore?

Reply:
@gl-sergei FYI ^

Reply:
`--reap` returned without an error with `send CREATE PROCEDURE p1() SELECT 1`.

Reply:
Hence my question :)

Reply:
`LOCK TABLES FOR BACKUP` doesn't work for `CREATE PROCEDURE` and `CREATE FUNCTION` anymore as `mysql.proc` (MyISAM) was removed in 8.0 and replaced with `information_schema.routines` (InnoDB).

Reply:
OK, We have discussed that's it's OK to allow such kinds of DDL under LTFB